R/plot-individual-time-profile.R

```diff
@@ -0,0 +1,118 @@
+#' Concentration time profile plot for individual
+#'
+#' @param dataCombined A `DataCombined` object.
+#' @param defaultPlotConfiguration A `DefaultPlotConfiguration` object, which is
+#'   an `R6` class object that defines plot properties.
+#'
+#' @import tlf
+#'
+#' @family plotting
+#'
+#' @examples
+#'
+#' # TODO: add example
```

Comment:
Please create an issue for `plotIndividualTimeProfile` documentation

Reply:
https://github.com/Open-Systems-Pharmacology/OSPSuite-R/issues/927